portable/ARMv8M/non_secure/portable/GCC/ARM_CM23/portmacro.h

```diff
@@ -60,6 +60,14 @@
 #if ( configTOTAL_MPU_REGIONS == 16 )
     #error 16 MPU regions are not yet supported for this port.
 #endif
+
+#ifndef configENABLE_MVE
+    #define configENABLE_MVE    0
+#else
+    #if( configENABLE_MVE != 0 )
+        #error configENABLE_MVE must be left undefined, or defined to 0 for the Cortex-M23.
+    #endif
+#endif
```

Comment:
```suggestion
#ifndef configENABLE_MVE
    #define configENABLE_MVE    0
#elif ( configENABLE_MVE != 0 )
        #error configENABLE_MVE must be left undefined, or defined to 0 for the Cortex-M23.
#endif
```

Reply:
If this works can we do this instead of the nested `#else/#if`?

Reply:
Done.